qml/Launcher/Launcher.qml

```diff
@@ -366,17 +366,26 @@ FocusScope {
         }
     }
 
-    BackgroundBlur {
-        id: backgroundBlur
-        anchors.fill: parent
-        anchors.topMargin: root.inverted ? 0 : -root.topPanelHeight
-        visible: root.interactiveBlur && root.blurSource && drawer.x > -drawer.width
-        sourceItem: root.blurSource
-        blurRect: Qt.rect(panel.width,
-                          root.topPanelHeight,
-                          drawer.width + drawer.x - panel.width,
-                          height - root.topPanelHeight)
-        occluding: (drawer.width == root.width) && drawer.fullyOpen
+    Item {
+        clip: true
+        x: 0
+        y: drawer.y
+        width: drawer.width + drawer.x
+        height: drawer.height
+        BackgroundBlur {
+            id: backgroundBlur
+            x: 0
```

Comment:
x/y should default to zero already, maybe this is not needed?

Reply:
I believe I did it this way to make it rather obvious and explicit how things work rather than expecting eventual readers to understand it immediately.